### PR TITLE
net-mail/metamail: last rites

### PR DESCRIPTION
**Pull request by Schievel1**

### CODE REVIEW

File: net-misc/hylafaxplus/hylafaxplus-7.0.3-r1.ebuild [hunk 1]
```diff
@@ -34,7 +34,6 @@ DEPEND="
 "
```
Comment (thesamesam) [inline net-misc/hylafaxplus/hylafaxplus-7.0.3-r1.ebuild:34]: Please git mv for these.

**Comment by thesamesam** [comment]
Summary of changes I made, check out what I did & let me know if any questions:
1. Add Bug: references to commits
2. Rebased package.mask/package.use.mask changes (didn't apply)
3. Adjusted commit message for profiles/* changes
4. Revbumped as noted above

I'll send the email for you because non-devs can't send to gentoo-dev-announce.

**Comment by Schievel1** [comment]
This commit message was actually auto generated by [lr-commit-pmask](https://github.com/projg2/mgorny-dev-scripts/blob/master/README.rst#lr-commit-pmask) from mgorny-dev-scripts. 
If we decide that it should be different we should make a PR (or just tell him, this is probably a one-liner)

**Comment by thesamesam** [comment]
Granted editbugs

**Comment by Schievel1** [comment]
<3

**Pull request closed without merging**